website/docs/configuration/securing-access-to-the-dashboard.mdx

```diff
@@ -97,26 +97,14 @@ If you need to change the OIDC configuration after running `gitops upgrade`, upd
 
 ## Login via a cluster user account
 
-Before you login via the cluster user account, you need to generate a bcrypt hash for your chosen password and store it as a secret in Kubernetes. There are several different ways to generate a bcrypt hash, this guide uses an Alpine Docker image to generate one:
+Before you login via the cluster user account, you need to generate a bcrypt hash for your chosen password and store it as a secret in Kubernetes. There are several different ways to generate a bcrypt hash, this guide uses a go Docker image to generate one:
```

Comment:
incredibly minor nit, but I _think_ it should be Go rather than go

Reply:
Fixed, in all 4 places.